tests/baselines/reference/booleanLiteralTypes1.types

```diff
@@ -1,32 +1,32 @@
 === tests/cases/conformance/types/literal/booleanLiteralTypes1.ts ===
 type A1 = true | false;
->A1 : boolean

```

Comment:
@ahejlsberg can you weigh in on `true | false` and `boolean` being equivalent to `type B = true | false` ?